src/realm/parser/driver.hpp

```diff
@@ -624,6 +624,7 @@ class ParserDriver {
     }
 
     Mixed get_arg_for_index(std::string);
+    double get_arg_for_coordinate(std::string);
```

Comment:
Same as above, this applies to line 626 too... unless you need to clone the string from some other reasons.